terraform/shared/shared.tf.json.template.py

```diff
@@ -40,6 +40,15 @@
                 }
             }
         },
+        'aws_s3_bucket_public_access_block': {
+            bucket: {
+                'bucket': '${aws_s3_bucket.%s.id}' % bucket,
+                'block_public_acls': True,
+                'block_public_policy': True,
+                'ignore_public_acls': True,
+                'restrict_public_buckets': True
+            } for bucket in ('shared_cloudtrail', 'versioned', 'aws_config', 'logs')
```

Comment:
See if you can avoid listing the resource names using TF's `for_each` construct. There are examples of it in the existing TF config.

Reply:
Addressed during PL.